integration-tests/opentelemetry-jaeger-remote/src/test/java/io/quarkus/it/opentracing/OTelJaegerRemoteTest.java

```diff
@@ -0,0 +1,131 @@
+package io.quarkus.it.opentracing;
```

Comment:
Sorry but this is opentelemetry, not opentracing.

Reply:
Will change this